mini_racer.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "minitest", "~> 5.0"
   spec.add_development_dependency "rake-compiler"
 
-  spec.add_dependency 'libv8', '~> 5.0'
+  spec.add_dependency 'libv8', '~> 5.1.281.59.0beta3'
```

Comment:
The version constraint should probably be more strict. How much more strict - I have no idea.